links/src/main/java/juja/microservices/links/controller/LinksController.java

```diff
@@ -1,20 +1,45 @@
 package juja.microservices.links.controller;
 
+import juja.microservices.links.exceptions.ApiErrorMessage;
+import juja.microservices.links.model.Link;
 import juja.microservices.links.model.SaveLinkRequest;
+import juja.microservices.links.service.LinksService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
 
+import javax.validation.Valid;
+
+@Slf4j
 @RestController
+@ResponseBody
 @RequestMapping(value = "/v1/links")
 public class LinksController {
+    private LinksService linksService;
+    private ApiErrorMessage errorMessage;
```

Comment:
it looks strange.

I think we don't need component for ApiErrorMessage
Spring has internal exceptions handler and we use it in our old services.
please use this implementation. 

For example - https://github.com/JujaLabs/teams-slack-bot/tree/master/src/main/java/ua/com/juja/microservices/teams/slackbot/exceptions

Reply:
done